RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -2257,6 +2275,33 @@
 			path = Previews;
 			sourceTree = "<group>";
 		};
+		2C2AEB402CA72A3000A50F38 /* PackageGroup */ = {
+			isa = PBXGroup;
+			children = (
+				2C2AEB4B2CA7339200A50F38 /* PackageComponentView.swift */,
+				2C2AEB452CA7302400A50F38 /* PackageComponentViewModel.swift */,
+			);
+			path = PackageGroup;
+			sourceTree = "<group>";
+		};
+		2C2AEB412CA72A3600A50F38 /* Package */ = {
+			isa = PBXGroup;
+			children = (
+				2C2AEB492CA7338A00A50F38 /* PackageGroupComponentView.swift */,
+				2C2AEB432CA72D9F00A50F38 /* PackageGroupComponentViewModel.swift */,
```

Comment:
Are these in the right place? Looks like `Package` contains `PackageGroup*` things, and vice versa.

Reply:
Ah no, I can see in the GitHub file tree that they're not in the right place 😄

Reply:
I see it's fixed in https://github.com/RevenueCat/purchases-ios/pull/4324. Nvm!